### DISCUSSION TITLE
[Feature]OCR search function by recognizing image text

### DISCUSSION THREAD

**Discussion by blogj** [immich-app/immich#12696] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [ ] Yes

### The feature

Is there any plan to enhance search through OCR recognition of image text? If it conflicts with the existing image recognition, a switch can be added to switch it.

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
#3168

As requested in the templates, please search for duplicates before opening a request.

**Comment by blogj** [comment]
Hello developers, will the OCR text recognition search function be included in the roadmap?

**Comment by elmocito** [comment]
Can't wait for OCR.
